src/fast_check/swc_helpers.rs

```diff
@@ -1,26 +1,17 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
-use deno_ast::swc::ast::Expr;
-use deno_ast::swc::ast::Ident;
-use deno_ast::swc::ast::Lit;
-use deno_ast::swc::ast::MemberProp;
-use deno_ast::swc::ast::Prop;
-use deno_ast::swc::ast::PropName;
-use deno_ast::swc::ast::PropOrSpread;
-use deno_ast::swc::ast::ReturnStmt;
-use deno_ast::swc::ast::Stmt;
-use deno_ast::swc::ast::TsEntityName;
-use deno_ast::swc::ast::TsKeywordType;
-use deno_ast::swc::ast::TsKeywordTypeKind;
-use deno_ast::swc::ast::TsLit;
-use deno_ast::swc::ast::TsLitType;
-use deno_ast::swc::ast::TsTupleElement;
-use deno_ast::swc::ast::TsType;
-use deno_ast::swc::ast::TsTypeAnn;
-use deno_ast::swc::ast::TsTypeOperator;
-use deno_ast::swc::ast::TsTypeOperatorOp;
-use deno_ast::swc::ast::TsTypeRef;
+use std::rc::Rc;
+
+use deno_ast::swc::ast::*;
```

Comment:
Did a bit of cleanup in this PR at the same time. Generally we don't use *, but I think it's ok in this module.